Improvements 2.x

dereuromark opened the pull request:
- added menuClass and wrapperClass for Bootstrapstyling
- made it a little bit more flexible.

torifat commented on View/Helper/MenuBuilderHelper.php:
```diff
@@ -261,9 +266,37 @@ protected function _buildItem(&$item, $pos = -1, &$isActive = false) {
+			$url = sprintf($this->settings['noLinkFormat'], __($item['title']));
 		} else {
-			$url = '<a title="' . $item['title'] . '" href="' . $this->Html->url($item['url']) . '">' . $item['title'] . '</a>';
+			$item['url'] += array('plugin' => false);
```
This line throws `PHP Fatal error:  Unsupported operand types in` during testing.

torifat replied:
@dereuromark can you please have a look?

dereuromark commented:
Closing until the reason for the travis fails can be found and resolved.

Commit:
added menuClass and wrapperClass for Bootstrapstyling
added images to menu
can be used at any level of the menu
Updated menu builder
added:
- absolute links
- link classes
- link target
Simplify merges
